package.json

```diff
@@ -16,7 +16,8 @@
     "test": "node --harmony_proxies node_modules/.bin/jest --coverage --config=jest.config.json",
     "stack:docker": "docker run --rm -it -p 8080:8080 -v \"$(pwd)/build\":/data/cozy-app/photos cozy/cozy-app-dev",
     "watch": "NODE_ENV=development webpack --config webpack.config.js --watch --display-modules --display-chunks",
-    "watch:standalone": "NODE_ENV=development webpack-dev-server --config webpack.config.js --port 8084 --display-modules --display-chunks --inline --hot"
+    "watch:standalone": "NODE_ENV=development webpack-dev-server --config webpack.config.js --port 8084 --display-modules --display-chunks --inline --hot",
+    "watch:test": "node --harmony_proxies node_modules/.bin/jest --watch --config=jest.config.json"
```

Comment:
With node 8.2.1, there is no such flag `--harmony_proxies`.
And we could reuse the `test` task to define `watch:test`:

```json
{
  "scripts": {
    "test": "node_modules/.bin/jest --coverage --config=jest.config.json",
    "watch:test": "npm run test -- --watch"
  }
```